packages/specs/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@eth-optimism/specs",
+  "version": "0.1.0",
+  "license": "MIT",
+  "devDependencies": {
+    "prettier": "2.2.1"
+  },
+  "scripts": {
+    "format": "yarn prettier --write ./**/*.md",
+    "format:check": "yarn prettier --check ./**/*.md"
```

Comment:
1. I intend for `format:check` to run in CI, but haven't set that up yet. 

2. I know that `yarn prettier --check ./**/*.md` looks funny. Why not just `prettier --check **/*.md`, honestly I'm not sure why, but that and every other variation of this script I tried failed to run on all md files up and down the tree.